pkg/model/master_volumes.go

```diff
@@ -282,6 +282,9 @@ func (b *MasterVolumeBuilder) addGCEVolume(c *fi.ModelBuilderContext, name strin
 
 func (b *MasterVolumeBuilder) addOpenstackVolume(c *fi.ModelBuilderContext, name string, volumeSize int32, zone string, etcd kops.EtcdClusterSpec, m kops.EtcdMemberSpec, allMembers []string) error {
 	volumeType := fi.StringValue(m.VolumeType)
+	if volumeType == "" {
```

Comment:
I do not understand this change, why we should default to tripleo? That can exist in redhat openstack, but not in another openstacks.

Reply:
instead if volumetype is empty, we should somehow detect that from openstack apis

Reply:
@zetaab i didn't take into consideration that other openstack flavors won't default to tripleo. It should be fine to just pass volume type in the cluster create command. I'll close this PR. Thank you for reviewing.